soundfile.py

```diff
@@ -849,12 +993,43 @@ def seek(self, frames, whence=SEEK_SET):
         _error_check(self._errorcode)
         return position
 
-    def tell(self):
+    def tell(self) -> int:
         """Return the current read/write position."""
         return self.seek(0, SEEK_CUR)
 
-    def read(self, frames=-1, dtype='float64', always_2d=False,
-             fill_value=None, out=None):
+
```

Comment:
see above

Reply:
Fixed in https://github.com/bastibe/python-soundfile/pull/462/commits/29c215fb43d8b7640f8634f68586ef07dbb79294